relentless/potential/potential.py

```diff
@@ -164,7 +164,7 @@ def evaluate(self, pair):
                 raise ValueError('Parameter {} is not set for ({},{}).'.format(p,pair[0],pair[1]))
 
             # evaluate the variable
-            if isinstance(v, Variable):
+            if isinstance(v, DesignVariable):
```

Comment:
This should revert to the base class `Variable`, since we now support evaluating any type.
```suggestion
            if isinstance(v, Variable):
```